crates/contracts/src/array_tr_storage/build_witness.rs

```diff
@@ -4,7 +4,7 @@ use simplicityhl::types::UIntType;
 use simplicityhl::value::{UIntValue, ValueConstructible};
 use simplicityhl::{WitnessValues, str::WitnessName};
 
-pub const MAX_VAL: usize = 10000; // u16::MAX as usize;
+pub const MAX_VAL: usize = 10000;
```

Comment:
In 4f0e1e976c3dfd28f1bcd7917cf78045f5cfcf96:

This should have been squashed into 882d8952062d19ba81d39e470991b3cbf03ebb99